src/h5web/visualizations/shared/models.ts

```diff
@@ -18,10 +18,18 @@ export interface DataAxisConfig {
 
 export type AxisConfig = IndexAxisConfig | DataAxisConfig;
 
-export interface AxisScale {
-  scale: DataScale;
-  scaleFn: DataScaleFn;
+export type AxisScaleFn = typeof scaleSymlog | typeof scaleLinear;
+
+export type AxisScale =
+  | ScaleLinear<number, number>
+  | ScaleSymLog<number, number>;
+
+export interface AxisInfo {
+  isIndexAxis: boolean;
+  scaleFn: AxisScaleFn;
   domain: Domain;
+  isLog: boolean;
+  showGrid: boolean;
```

Comment:
The new `AxisInfo` interface.

Reply:
I wonder if `isLog` is still necessary in `AxisInfo` given that its purpose was to choose which scale function to pick and it is already done as this point.
That being said, I find it clearer to keep it..

Reply:
Ha true!

>That being said, I find it clearer to keep it..

As you wish 👍